apps/condo/pages/auth/forgot.tsx

```diff
@@ -110,65 +106,74 @@ function ResetPageView () {
             setIsLoading(false)
         })
     }
+    const Content = () => {
+        if (isLoading) {
+            return <Loader size="large" />
+        }
+        return (
+            <>
+                <Col span={24}>
+                    <Typography.Title style={{ textAlign: 'left' }}>{ResetTitle}</Typography.Title>
+                </Col>
+                <Col span={24}>
+                    <Typography.Paragraph style={{ textAlign: 'left' }}>{InstructionsMsg}</Typography.Paragraph>
+                </Col>
+                <Col span={24}> 
+                    <Form
+                        {...FORM_LAYOUT}
+                        form={form}
+                        name='forgot-password'
+                        validateTrigger={['onBlur', 'onSubmit']}
+                        initialValues={initialValues}
+                        colon={false}
+                        labelAlign='left'
+                        requiredMark={false}
+                    >
+                        <Row gutter={[0, 60]}>
+                            <Col span={24}>
+                                <Form.Item
+                                    name='phone'
+                                    label={PhoneMsg}
+                                    rules={validations.phone}
+                                >
+                                    <PhoneInput placeholder={ExamplePhoneMsg} />
+                                </Form.Item>
+                            </Col>
+                            <Col span={24}>
+                                <Form.Item>
+                                    <CountDownTimer action={startConfirmPhoneAction} id={'FORGOT_ACTION'} timeout={LOCK_TIMEOUT}>
+                                        {({ countdown, runAction }) => {
+                                            const isCountDownActive = countdown > 0
+                                            return (
+                                                <Button
+                                                    onClick={() => {
+                                                        form.validateFields().then(() => {
+                                                            runAction()
+                                                        }).catch(_ => {
+                                                        // validation check failed - don't invoke runAction
+                                                        })
```

Comment:
Maybe to write something to console here, it will be sent to Sentry for further debugging